distribution/src/resources/drill-override-example.conf

```diff
@@ -184,7 +184,12 @@ drill.exec: {
       # Set this property if custom absolute root should be used for remote directories
       root: "/app/drill"
     }
-  }
+  },
+  # Setting for Temporary Tables.
+  # See https://gist.github.com/arina-ielchiieva/50158175867a18eee964b5ba36455fbf#file-temporarytablessupport-md.
+  # Temporary table can be created ONLY in default temporary workspace.
+  # Full workspace name should be indicated (including schema and workspace separated by dot). Workspace MUST be writable.
+  default_temporary_workspace: "dfs.tmp"
```

Comment:
Any restrictions here on which workspace can be used? Must it be a file-based workspace? (Probably can't be a JDBC storage plugin, right?) Do we check anywhere that the plugin referenced here is of the correct type?